include/deal.II/lac/affine_constraints.h

```diff
@@ -1961,24 +1961,24 @@ class AffineConstraints : public Subscriptor
    *
    * This method checks if all processors agree on the constraints for their
    * local lines as given by @p locally_active_dofs. This method is a collective
-   * operation and will return @p true only if all processors are consistent.
+   * operation and will return `true` only if all processors are consistent.
    *
    * Please supply the owned DoFs per processor as returned by
-   * Utilities::MPI::all_gather(MPI_Comm, DoFHandler::locally_owned_dofs()) as
+   * `Utilities::MPI::all_gather(MPI_Comm, DoFHandler::locally_owned_dofs())` as
    * @p locally_owned_dofs and the result of
-   * DoFTools::extract_locally_active_dofs() as
+   * `DoFTools::extract_locally_active_dofs()` as
```

Comment:
I don't think this markup is necessary, this way it is a link to the function.

Reply:
Indeed. If doxygen recognizes something as a link to another function, it will automatically use the right font.

Reply:
You are right, see [current documentation](https://www.dealii.org/developer/doxygen/deal.II/classAffineConstraints.html#ad5fb8d3ca686a76ba33eb82d7f52b5fd). I will adjust the patch.